app/models/saved_claim.rb

```diff
@@ -30,6 +30,9 @@ class SavedClaim < ApplicationRecord
   has_many :form_submissions, dependent: :nullify
   has_many :claim_va_notifications, dependent: :destroy
 
+  has_one :appeal_submission, class_name: 'AppealSubmission', foreign_key: :submitted_appeal_uuid, primary_key: :guid,
```

Comment:
Can we move this to our `SavedClaim` models? Specifically into `SavedClaim::HigherLevelReview`, `SavedClaim::NoticeOfDisagreement` and `SavedClaim::SupplementalClaim` instead of the base `SavedClaim`.

Reply:
OK, changed the approach, let me know if this looks good